packages/calcite-components/src/assets/styles/_floating-ui.scss

```diff
@@ -26,11 +26,11 @@ $floating-ui-transform-right: translateX(-5px);
 
 @mixin floatingUIElemAnim(
   $selector,
-  $boxShadow: "--calcite-shadow-md",
-  $borderRadius: "--calcite-corner-radius-0",
-  $borderWidth: "--calcite-border-width-none",
-  $borderColor: "--calcite-color-border-3",
-  $backgroundColor: "--calcite-color-foreground-1"
+  $boxShadow: "var(--calcite-shadow-md)",
+  $borderRadius: "var(--calcite-corner-radius-0)",
+  $borderWidth: "var(--calcite-border-width-none)",
+  $borderColor: "var(--calcite-color-border-3)",
+  $backgroundColor: "var(--calcite-color-foreground-1)"
```

Comment:
should this be refactored for tokens? Something like `--calcite-floating-el-[shadow, corner-radius, border-color, background-color]`?

Reply:
These are just the default values, components using this mixin would pass in their tokens here.

`--calcite-floating-el` tokens don't make sense because its not actually a component.